### ISSUE TITLE
Module ctc.evm has not attribute(s)

### ISSUE THREAD

**Issue by steelep** [fei-protocol/checkthechain#28]
Hi, 

Attempting run some basic queries (these directly are from the doc example pages): 

e.g. 
```
from ctc import evm
  contract_address = '0xd8da6bf26964af9d7eed9e03e53415d37aa96045'
  transactions = await evm.async_get_transactions_from_address(address)
  len(transactions)
```

results in:
```
> ---------------------------------------------------------------------------
> AttributeError                            Traceback (most recent call last)
> /tmp/ipykernel_3288/4106459712.py in async-def-wrapper()
>       4 len(transactions)
>       5 
> ----> 6 
> 
> AttributeError: module 'ctc.evm' has no attribute 'async_get_transactions_from_address'
```

While calling other evm.async_X methods work absolutely fine, is something wrong with the namespace?

e.g. 

```
from ctc import evm

dao_address = '0x0bef27feb58e857046d630b2c03dfb7bae567494'

proposals = await evm.async_get_events(
    contract_address=dao_address,
    event_name='ProposalCreated',
)
```
returns as expected

Any ideas here? I have tried installing via pip and building directly from source (at current repo state) - same odd result for both routes.

Thanks - I love the tooling here.

**Comment by sslivkoff** [comment]
the function `async_get_transactions_from_address` is in the upcoming 0.3.0 release

the version of ctc on pypi is only 0.2.10, so it doesnt have it yet

which commit did you build from? building from latest commit should have it. maybe make sure the older pypi version is totally removed?

**Comment by Skytokenn** [comment]
Kindly open this live chat link to talk to the customer service directly on your issue  https://direct.lc.chat/14334663/

**Comment by sslivkoff** [comment]
now resolved with ctc 0.3.0
